tests/test_session.py

```diff
@@ -2,8 +2,6 @@
 
 
 def test_view_sessions(client, auth):
-    # getting the form as a logged in users,
-    # reguarless if the user owns the course or not
     auth.login()
```

Comment:
we may want to add the comments back in explaining what the first part of the test is doing